src/_data/groups.yml

```diff
@@ -133,6 +160,19 @@
   website: http://columbusrb.com/
   email: board@columbusrb.com
 
+- name: fleur de ruby
+  source:
```

Comment:
I think the source being blank might throw off:

https://github.com/MikeRogers0/Ruby-Meetup-Calendar/blob/61a95606e71026c9cc03b6d99a43da79a7378c6c/lib/posts_generator.rb#L4-L8

We'll want to update that LOC to filter out empty sources.